swift_browser_ui/health.py

```diff
@@ -120,6 +132,12 @@ async def get_upload_runner(
         services["swiftui-upload-runner"] = {
             "status": "Error"
         }
+    except Exception as e:
+        request.app["Log"].debug("Poll swift-x-account-sharing failed")
+        request.app["Log"].info(f"reason: {e}")
+        services["swift-sharing-request"] = {
+            "status": "Error"
+        }
```

Comment:
Imo this would make a good function, since the exact same exception is here multiple times

Reply:
fair enough will add that.